docs-markdown/package.json

```diff
@@ -214,6 +219,11 @@
           "command": "applyTemplate",
           "group": "Docs",
           "when": "vscode.extensions.getExtension('docsmsft.docs-article-templates').active"
+        },
+        {
+          "command": "applyCleanup",
+          "group": "Docs",
+          "when": "vscode.extensions.getExtension('docsmsft.docs-article-templates').active"
```

Comment:
I think we can change this to "resourceLangId == markdown" so it doesn't have a dependency on the templates extension.